### PR TITLE
Fixed #3498. Show missing pixmaps in VeSTige plugin.

### LINKED ISSUE #3498: VeSTige preset left and right arrows are gone
In VeSTige instrument left and right arrows to browse presets are missing. Instead it shows something that looks like dead pixels. I remember that I was using these arrows about month ago on master branch.

screenshot_20170411_004313

This picture from Wiki shows how it used to look:

LMMS 1.2.0-rc2.144
(Linux x86_64, Qt 5.6.1, GCC 6.2.0 20161005)

### PR DESCRIPTION
**Pull request by karmux**

### CODE REVIEW

**Comment by tresf** [comment]
@karmux thanks!  I'm going to ask @Lukas-W to merge this since he rewrote the resource system.

File: plugins/vestige/vestige.cpp [hunk 1]
```diff
@@ -259,7 +259,7 @@ void vestigeInstrument::loadFile( const QString & _file )
 	TextFloat * tf = TextFloat::displayMessage(
 			tr( "Loading plugin" ),
 			tr( "Please wait while loading VST-plugin..." ),
-			PLUGIN_NAME::getIconPixmap( "logo", 24, 24 ), 0 );
+			embed::getIconPixmap( "logo", 24, 24 ), 0 );
```
Comment (Lukas-W) [inline plugins/vestige/vestige.cpp]: I don't think this and other lines loading `logo.png` should change, as they're about loading the plugin's logo, not a shared resource, right?

Comment (karmux) [inline plugins/vestige/vestige.cpp]: Right. I reverted logo changes.

Comment (Lukas-W) [inline plugins/vestige/vestige.cpp]: Thanks!

File: plugins/vestige/vestige.cpp [hunk 2]
```diff
@@ -534,7 +534,7 @@ VestigeInstrumentView::VestigeInstrumentView( Instrument * _instrument,
 	QPushButton * note_off_all_btn = new QPushButton( tr( "Turn off all "
 							"notes" ), this );
 	note_off_all_btn->setGeometry( 20, 160, 200, 24 );
-	note_off_all_btn->setIcon( embed::getIconPixmap( "state_stop" ) );
+	note_off_all_btn->setIcon( embed::getIconPixmap( "stop" ) );
```
Comment (zonkmachine) [inline plugins/vestige/vestige.cpp]: This part of the fix should be in stable-1.2

**Pull request merged**